spec/forms/personal_info_form_spec.rb

```diff
@@ -29,6 +30,21 @@
       end
     end
 
+    context "if we're trying to .save data that cannot make it into the db (mismatch between model and form validation)" do
+      it "creates neither a client nor an intake" do
+        weird_params = valid_params.merge(additional_params).merge(phone_number: '0000000000', phone_number_confirmation: '0000000000')
```

Comment:
i thought this example was no longer a mismatch between the model and form validation?